masl.src.html

```diff
@@ -174,7 +174,7 @@ <h3>HTTP Headers</h3>
         <p>
           MASL supports a subset of HTTP response headers that are meaningful in decentralised
           contexts. This doesn't preclude headers not listed here from being used, but implementations
-          that support using HTTP headers SHOULD NOT reflect the value of arbitrary HTTP headers
+          that support using HTTP headers SHOULD NOT pass through values for additional HTTP headers
```

Comment:
```suggestion
```
No, "reflect" is a term of art and it indicates that implementations can be conformant if they pass a semantically equivalent value but specifically don't pass it through byte for byte.